elixir/apps/domain/lib/domain/billing/stripe/api_client.ex

```diff
@@ -93,6 +93,7 @@ defmodule Domain.Billing.Stripe.APIClient do
       URI.encode_query(
         %{
           "customer" => customer_id,
+          "automatic_tax" => %{"enabled" => true},
```

Comment:
If these are query params, how do you encode a nested object? Sure this isn't meant to be `automatic_tax=true`?

Reply:
Hm good catch, I assumed this was using Plug.Conn.Query.encode which supports nested maps.

Updated.

Reply:
Yeah that's the field structure according to docs.